ui/desktop/src/main.ts

```diff
@@ -315,10 +315,26 @@ app.whenReady().then(async () => {
 
   Menu.setApplicationMenu(menu);
 
-  app.on('activate', () => {
+  app.on('activate', async () => {
     if (BrowserWindow.getAllWindows().length === 0) {
       createChat(app);
     }
+    const sources = await electron.desktopCapturer.getSources({ types: ['screen'] })
```

Comment:
this is to try to trigger a request for permissions early

Reply:
Does it req the permissions right when the app launches?

Users will be more primed to accept if it happens when they try to do something that needs the access.

Reply:
hrm - will at both in theory. I need to play around with it more to be sure. I can get rid of this so that it asks later